pyccel/codegen/printing/ccode.py

```diff
@@ -666,14 +688,19 @@ def _print_NumpyRandint(self, expr):
 
     def _print_FunctionDef(self, expr):
 
+        if len(expr.results) > 1:
+            self._additional_args.append(expr.results)
```

Comment:
I've just realized that `_additional_args` is a list of lists. Do you need this to handle nested functions?

Reply:
Yes. A list was used, but I recommended using a list of lists for nested functions